src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -108,7 +111,7 @@
 @end
 
 @private processLoopView(view)
-  for ({@view.variable} in {@variableField(view.collection)}):
+  for {@view.variableName} in {@variableField(view.collection)}:
```

Comment:
in `processAssignmentView`, could you also make s/`variable`/`variableName`/? My mistake!